src/hotspot/share/classfile/symbolTable.cpp

```diff
@@ -609,15 +607,6 @@ void SymbolTable::write_to_archive(GrowableArray<Symbol*>* symbols) {
   if (!DynamicDumpSharedSpaces) {
     _shared_table.reset();
     writer.dump(&_shared_table, "symbol");
-
-    // Verify the written shared table is correct -- at this point,
-    // vmSymbols has already been relocated to point to the archived
-    // version of the Symbols.
-    Symbol* sym = vmSymbols::java_lang_Object();
-    const char* name = (const char*)sym->bytes();
-    int len = sym->utf8_length();
-    unsigned int hash = hash_symbol(name, len, _alt_hash);
-    assert(sym == _shared_table.lookup(name, hash, len), "sanity");
```

Comment:
Maybe keeping this within a DEBUG_ONLY block if it's still applicable?

Reply:
This check cannot be done anymore -- `_shared_table` is initialized with the "requested" addresses but `sym`is still at "buffer" address, so the check will fail (I forgot whether the lookup would crash or the assert will fail).